shared/stories/index.desktop.js

```diff
@@ -12,6 +12,17 @@ import {initDesktopStyles} from '../styles/index.desktop'
 
 const stories = {...sharedStories, ...desktopStories}
 
+const filter = null
+const filteredStories = Object.keys(stories).reduce(
```

Comment:
can we just load this off the query param in window, then we don't need to do this in code?

Reply:
(or env var or something)